src/lib/core/gestures/MdGestureConfig.ts

```diff
@@ -4,16 +4,25 @@ import {HammerGestureConfig} from '@angular/platform-browser';
 /* Adjusts configuration of our gesture library, Hammer. */
 @Injectable()
 export class MdGestureConfig extends HammerGestureConfig {
+  private _hammer = typeof window !== 'undefined' ? (window as any).Hammer : null;
 
   /* List of new event names to add to the gesture support list */
-  events: string[] = [
+  events: string[] = this._hammer ? [
     'longpress',
     'slide',
     'slidestart',
     'slideend',
     'slideright',
     'slideleft'
-  ];
+  ] : [];
+
+  constructor() {
+    super();
+
+    if (!this._hammer) {
```

Comment:
Can you import `isDevMode` from `@angular/core` and then only log this warning if `isDevMode()` is true?